src/databricks/labs/remorph/helpers/recon_config_utils.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+from databricks.labs.blueprint.tui import Prompts
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.remorph.helpers.db_workspace_utils import DatabricksSecretsClient
+from databricks.labs.remorph.reconcile.constants import SourceType
+
+logger = logging.getLogger(__name__)
+
+recon_source_choices = [
+    SourceType.SNOWFLAKE.value,
+    SourceType.ORACLE.value,
+    SourceType.DATABRICKS.value,
+    SourceType.NETEZZA.value,
+]
+
+
+class ReconConfigPrompts:
+    def __init__(self, ws: WorkspaceClient, prompts: Prompts = Prompts()):
+        self._source = None
+        self._prompts = prompts
+        self._db_secrets = DatabricksSecretsClient(ws, prompts)
```

Comment:
move all methods from this class over here

Reply:
Done